doc/classes/OS.xml

```diff
@@ -176,91 +195,108 @@
-				Similar to [method get_cmdline_args], but this returns the user arguments (any argument passed after the double dash [code]--[/code] or double plus [code]++[/code] argument). These are left untouched by Godot for the user. [code]++[/code] can be used in situations where [code]--[/code] is intercepted by another program (such as [code]startx[/code]).
-				For example, in the command line below, [code]--fullscreen[/code] will not be returned in [method get_cmdline_user_args] and [code]--level 1[/code] will only be returned in [method get_cmdline_user_args]:
+				Returns the command-line user arguments passed to the engine. User arguments are ignored by the engine and reserved for the user. They are passed after the double dash [code]--[/code] argument. [code]++[/code] may be used also used when [code]--[/code] is intercepted by another program (such as [code]startx[/code]).
 				[codeblock]
-				godot --fullscreen -- --level 1
-				# Or:
-				godot --fullscreen ++ --level 1
+				# Godot has been executed with the following command:
+				# godot --fullscreen -- --level=2 --hardcore 
+
+				OS.get_cmdline_args()      # Returns ["--fullscreen", "--level=2", "--hardcore"]
+				OS.get_cmdline_user_args() # Returns ["--level=2", "--hardcore"]
 				[/codeblock]
+				To get all passed arguments, use [method get_cmdline_args].
 			</description>
 		</method>
 		<method name="get_config_dir" qualifiers="const">
 			<return type="String" />
 			<description>
-				Returns the [i]global[/i] user configuration directory according to the operating system's standards. On the Linux/BSD platform, this path can be overridden by setting the [code]XDG_CONFIG_HOME[/code] environment variable before starting the project. See [url=$DOCS_URL/tutorials/io/data_paths.html]File paths in Godot projects[/url] in the documentation for more information. See also [method get_cache_dir] and [method get_data_dir].
+				Returns the [i]global[/i] user configuration directory according to the operating system's standards.
+				On the Linux/BSD platform, this path can be overridden by setting the [code]XDG_CONFIG_HOME[/code] environment variable before starting the project. See [url=$DOCS_URL/tutorials/io/data_paths.html]File paths in Godot projects[/url] in the documentation for more information. See also [method get_cache_dir] and [method get_data_dir].
 				Not to be confused with [method get_user_data_dir], which returns the [i]project-specific[/i] user data path.
 			</description>
 		</method>
 		<method name="get_connected_midi_inputs">
 			<return type="PackedStringArray" />
 			<description>
-				Returns an array of MIDI device names.
-				The returned array will be empty if the system MIDI driver has not previously been initialized with [method open_midi_inputs].
+				Returns an array of MIDI device names. The returned array will be empty if the system MIDI driver has not previously been initialized with [method open_midi_inputs]. See also [method close_midi_inputs].  
```

Comment:
There are 2 spaces at the end of this line.

```suggestion
				Returns an array of MIDI device names. The returned array will be empty if the system MIDI driver has not previously been initialized with [method open_midi_inputs]. See also [method close_midi_inputs].
```

Reply:
Interesting. The checks didn't automatically detect this.